launch/launch/actions/execute_local.py

```diff
@@ -184,7 +184,7 @@ def __init__(
             Use get_stdout() and get_stderr() to read the buffered output.
         :param: on_exit list of actions to execute upon process exit.
         :param: respawn if 'True', relaunch the process that abnormally died.
-            Defaults to 'False'. If Substitution is given, it will be resolved at runtime.
+            Either a boolean of a Substitution to be resolved at runtime. Defaults to 'False'.
```

Comment:
@kosuke55 nit: 
```suggestion
            Either a boolean or a Substitution to be resolved at runtime. Defaults to 'False'.
```

Reply:
Oh, I fixed in https://github.com/ros2/launch/pull/569/commits/80bbce6a56b0a79695f3a7637fe81742b6965239